pages/pix-site/index.vue

```diff
@@ -1,14 +1,43 @@
-<template><div></div></template>
+<template>
+  <client-only v-if="shouldDisplayLocaleChoice">
+    <main class="home">
+      <div class="locale-choice">
+        <h1>
+          <pix-image
+            class="logo-pix"
+            :field="{ url: '/images/logo-pix-blanc.svg', alt: 'Pix' }"
+          />
+        </h1>
+        <LocaleLink
+          v-for="locale in locales"
+          :key="locale.code"
+          class="locale-choice__link"
+          :locale="locale"
+        />
+        <pix-image class="planet" :field="{ url: '/images/planet.svg' }" />
+      </div>
+    </main>
+  </client-only>
+</template>
 
 <script>
+import { language } from '~/config/language'
 export default {
+  layout: 'empty',
   nuxtI18n: false,
-  layout: 'simple',
+  data() {
+    return {
+      locales: language.locales,
+      shouldDisplayLocaleChoice: false,
+    }
+  },
   mounted() {
     const preferredLocale = this.getLocaleFromCookie()
-    document.location = preferredLocale
-      ? `/${preferredLocale}/`
-      : '/locale-choice'
+    if (preferredLocale) {
+      document.location = preferredLocale
```

Comment:
Incertitude de si c'est mieux d'utiliser la redirection "dure" ou un `this.$router.push`

Reply:
On préfère utiliser `$router.push` comme précisé dans la section "Redirection navigateur vs. History API" de [cette PR](https://github.com/1024pix/pix-site/pull/455).